lib/praxis/action_definition.rb

```diff
@@ -188,32 +188,71 @@ def description(text = nil)
       @description
     end
 
+    def self.url_description(route:, params_example:, params:)
+      route_description = route.describe
+
+      example_hash = params_example ? params_example.dump : {}
+      hash = self.url_example(route: route, example_hash: example_hash, params: params)
+
+      query_string = URI.encode_www_form(hash[:query_params])
+      url = hash[:url]
+      url = [url,query_string].join('?') unless query_string.empty?
+
+      route_description[:example] = url
+      route_description
+    end
+
+    def self.url_example(route:, example_hash:{}, params:)
```

Comment:
This method is also copied in response.rb (they are identical).
It is a class-level method helper already, so let's move it somewhere and have the callers calls use it.